src/com/mambu/apisdk/services/CommentsService.java

```diff
@@ -0,0 +1,142 @@
+import com.mambu.core.shared.model.Comment;
+
+/**
+ * Service class which handles API operations for retrieving and posting comments for Mambu Entities. Posting and
+ * Getting Comments for the following entities is currently supported: Client, Group. LoanAccount, SavingsAccount,
+ * LoanProduct, SavingsProduct, Branch, Centre, User
+ * 
+ * Comments API currently supports:
+ * 
+ * GET Comments (see MBU-8608 - As a Developer, I'd like to GET comments via APIs) and
+ * 
+ * POST Comment (see MBU-8609 - As a Developer, I need to POST comments via APIs)
+ * 
+ * @author mdanilkis
+ * 
+ */
+
+public class CommentsService {
+	// Specify Entity managed by the class
+	private final static MambuEntityType ownedEntity = MambuEntityType.COMMENT;
+
+	// Service helper
+	protected ServiceExecutor serviceExecutor;
+
+	// Specify Mambu entities supported by the Comments API
+	// Comments are supported by Client, Group. LoanAccount, SavingsAccount, LoanProduct, SavingsProduct, Branch,
+	// Centre, User
+	private final static MambuEntityType[] supportedEntities = new MambuEntityType[] { MambuEntityType.CLIENT,
+			MambuEntityType.GROUP, MambuEntityType.LOAN_ACCOUNT, MambuEntityType.SAVINGS_ACCOUNT,
+			MambuEntityType.LOAN_PRODUCT, MambuEntityType.SAVINGS_PRODUCT, MambuEntityType.BRANCH,
+			MambuEntityType.CENTRE, MambuEntityType.USER };
+
+	/***
+	 * Create a new service
+	 * 
+	 * @param mambuAPIService
+	 *            the service responsible with the connection to the server
+	 */
+	@Inject
+	public CommentsService(MambuAPIService mambuAPIService) {
+		this.serviceExecutor = new ServiceExecutor(mambuAPIService);
+	}
+
+	/**
+	 * Create new Comment. Only "text" field is used, other comment fields are ignored for new comments
+	 * 
+	 * @param parentEntity
+	 *            MambuEntityType for which comments are retrieved. Example: MambuEntityType.CLIENT for comments owned
+	 *            by Client. Must not be null. Comments for the following entities are currently supported: Client,
+	 *            Group. LoanAccount, SavingsAccount, LoanProduct, SavingsProduct, Branch, Centre, User
+	 * @param parentEntityId
+	 *            entity id or encoded key for the parent entity. Example, ciinetId for MambuEntityType.CLIENT. Must not
+	 *            be null
+	 * @param comment
+	 *            comment to post. Must not be null
+	 * @return created comment
+	 * @throws MambuApiException
+	 */
+	public Comment create(MambuEntityType parentEntity, String parentEntityId, Comment comment)
+			throws MambuApiException {
+
+		// POST {"comment:":{"text":"Posting a new comment" }} /api/centres/ABC123/comments
+		// Available since Mambu 3.11 See MBU-8609 for more details
+
+		if (comment == null) {
+			throw new IllegalArgumentException("Comment cannot be null");
+		}
+		// Set all fields except the text field to null: Mambu expects only on field in the JSON request
+		Comment postComment = new Comment();
+		postComment.setText(comment.getText());
```

Comment:
Please add this constraint the parameter description as well, ideally we could have this method just being a delegate to a method that just takes a String instead of a Comment object, to not suggest that a whole comment object needs to be populated before being sent to the server.